dead_mans_switch_client/models/dead_mans_switch_client.py

```diff
@@ -11,6 +11,8 @@
 import urllib2
 from openerp import api, models
 
+SEND_TIMEOUT = 60
```

Comment:
could you please make this a config parameter? Just say `self.env['ir.config_parameter'].get_param('dead_mans_switch_client.send_timeout`, 60)` below. Also drop a sentence about that in the readme